linux/test/test_services.sh

```diff
@@ -42,14 +42,14 @@ docker exec -it $CNAME /bin/bash -c 'd=10; \
         done'
 
 #check OMERO.server service status
-docker exec -it $CNAME /bin/bash -c "service omero status -l"
+docker exec -it $CNAME /bin/bash -c "systemctl status omero.service -l"
```

Comment:
if https://github.com/ome/ome-docker/pull/36/files is in the above can be reverted